src/main/java/com/minecolonies/core/entity/pathfinding/MNode.java

```diff
@@ -221,6 +221,14 @@ public void setVisited()
         visited = true;
     }
 
+    /**
+     * Sets the node as closed.
+     */
+    public void unSetVisited()
```

Comment:
hmm, method above has same impl, plus the same doesnt match

Reply:
indeed, bug :)